test/unit/src/audio/FftUnit.cpp

```diff
@@ -50,9 +50,11 @@ TEST_CASE( "audio/Fft" )
 
 SECTION( "round trip error" )
 {
+#if defined( CINDER_MAC )

```

Comment:
Could you move this define to the top, replacing [this line](https://github.com/lucasvickers/Cinder/blob/15b051deb7b044b6dd923f06bd87c5ab437269e5/test/unit/src/audio/FftUnit.cpp#L4)? I'm hoping to look at this soon but at least that will keep things tidy in the meantime.